models/lever.yml

```diff
@@ -87,12 +89,16 @@ models:
 
   - name: lever__opportunity_enhanced
     description: Table of unique opportunities.
-    columns:
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - source_relation 
+            - opportunity_id
+    columns: 
+      - name: source_relation
+        description: "{{ doc('source_relation') }}"
       - name: opportunity_id
         description: Unique ID of the opportunity.
-        tests:
-          - unique
-          - not_null
```

Comment:
Was this not_null intentionally removed? If so, we should add this to the CHANGELOG. Otherwise, we should add it back in as a test on this field.

Reply:
Same note for other removals